components/CategoriesBar/CategoryItem/CategoryItem.tsx

```diff
@@ -0,0 +1,21 @@
+import type { TranslatedCategory } from '@prezly/sdk';
+
+import { DropdownItem } from '@/components/Dropdown';
+
+import styles from './CategoryItem.module.scss';
+
+type Props = {
+    category: TranslatedCategory;
+};
+
+export function CategoryItem({ category }: Props) {
+    const { name } = category;
+    const description = 'Lorem ipsum dolor sit amet';
```

Comment:
lets remove the Lorem 😄

Reply:
Good catch, almost ended up like this:

![sg974cu8w7dc1](https://github.com/user-attachments/assets/7ca20669-48c6-4694-9294-660bab2e2d2a)